aptos-move/framework/aptos-stdlib/sources/type_info.move

```diff
@@ -18,6 +20,7 @@ module aptos_std::type_info {
     }
 
     public native fun type_of<T>(): TypeInfo;
+    public native fun full_name<T>(): string::String;
```

Comment:
maybe full_path? or just path?

Reply:
@wrwg opinion?

Reply:
Hmm... looks asymmetric. `type_info::name_of<T>()`?

Reply:
works for me and better than "path". originally I named it `type_info::type_name()`